src/plcc/load_spec/validate_spec/validate_syntactic_spec/validate_rhs_test.py

```diff
@@ -70,6 +72,133 @@ def test_invalid_Rhs_error():
     assert len(errors) == 1
     assert errors[0] == makeInvalidRhsNameFormatError(spec[1])
 
+def test_no_duplicate_Rhs_nonterminal():
```

Comment:
So, I'm making another PR update their tests to use the parser. And although I don't think these tests suffers from readability as much, they do suffer from duplicate code. We'll talk about this today.